Documentation-Requirements.md

```diff
@@ -33,6 +34,25 @@ If there are multiple notes, bullets should be used:
 >
 > - I am important point *n*.
 
+# Files and command names
+
+All filenames and command names should be rendered in a fixed-format font
+using backticks:
+
+- > Run the `foo` command to make it work.
+
+- > Modify the `bar` option in file `/etc/baz/baz.conf`.
```

Comment:
Having these first two notes bulleted, but the next single pair not looks a little odd on rendering. I'd say either bullet all four, or none?